specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/HybridLinkUpdate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "managedInstanceName": "testcl",
+    "distributedAvailabilityGroupName": "dag",
+    "api-version": "2020-11-01-preview",
+    "parameters": {
+      "properties": {
+        "replicationMode": "Sync"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "31ab7bb2-3233-4a74-9fab-7e72da2a541f",
+        "id": "/subscriptions/f2669dff-5f08-45dd-b857-b2a60b72cdc9/resourceGroups/testrg/providers/Microsoft.Sql/locations/onebox/hybridLinkOperationResults/31ab7bb2-3233-4a74-9fab-7e72da2a541f",
+        "type": "Microsoft.Sql/managedInstances/hybridLinkOperationResults"
+      }
+    },
+    "202": {
+      "operation": "UpdateCopyLinkReplicationMode",
+      "startTime": "2020-07-19T17:00:51.697Z"
```

Comment:
The schema for this is not defined. 202 should return an empty body with headers.